scripts/firedrake-install

```diff
@@ -142,6 +143,24 @@ Please follow the instructions at http://www.firedrakeproject.org/download.html
             print("""\nfiredrake-update must be run with Python 3, did you accidentally use Python 2?""")
     sys.exit(1)
 
+# Temporary catch for tigerlake processors with gcc/g++/gfortran 9.3
+# But only for Linux
+if platform.system() == "Linux":
+    # Get the cpu information
+    cpuinfostring = os.popen('lscpu').read()
+    # Check if it is a tigerlake processor
+    if cpuinfostring.find('i3-11') != -1 or cpuinfostring.find('i5-11') != -1 or cpuinfostring.find('i7-11') != -1 or cpuinfostring.find('i9-11') != -1:
+        # Get the compiler versions
+        gccversionstring = os.popen('gcc -dumpfullversion').read()
+
+        gppversionstring = os.popen('g++ -dumpfullversion').read()
+
+        gfortranversionstring = os.popen('gfortran -dumpfullversion').read()
+
+        # Check if the combination is a problem
+        if pversion.parse(gccversionstring) <= pversion.parse("9.3") or pversion.parse(gppversionstring) <= pversion.parse("9.3") or pversion.parse(gfortranversionstring) <= pversion.parse("9.3"):
```

Comment:
I think there might be a subtle logical flaw here. If I understand correctly, versions up to and including 9.4.x are a problem. However:
```ipython
In [1]: from packaging.version import parse

In [2]: parse("9.3.1") <= parse("9.3")
Out[2]: False
```
Think you might need `<parse("9.4")` instead of `<= parse("9.3")`.

Reply:
This is absolutely correct!